### DISCUSSION TITLE
Update the Contribution Guidelines, Code of Conduct, and Security Policy of Simplified Coding

### DISCUSSION THREAD

**Discussion by odysseuskir** [simplified-coding/simplified-coding#125] [Ιδέες / Σχόλια]
All three of these important documents have been poorly written or/and copied from other sources. I propose the team collectively works on rewriting these.

**Comment by GalaxyGamingBoy** [comment]
You literally chatgpted them

**Reply by odysseuskir** [reply]
Yup
